### PR TITLE
Add support for . version delimiter

### PR DESCRIPTION
**Pull request by fdplacido**
BEGINRELEASENOTES
- Add support for "." version delimiter

ENDRELEASENOTES

### CODE REVIEW

File: scripts/tagging/parseversion.py [hunk 1]
```diff
@@ -27,6 +29,12 @@ def __init__( self, versionString, makePreRelease=True ):
     self.isPre = False
     self.makePreRelease=makePreRelease
     self.log = getLogger( "Version" )
+    self.delimiter = ""
```
Comment (andresailer) [inline scripts/tagging/parseversion.py:32]:
```suggestion
    self.delimiter = '-' if '-' in self.version else '.'
```

**Comment by andresailer** [comment]
Please add a couple dotted version to the Test_version.py tests

**Comment by fdplacido** [comment]
I added the test and it passes.

File: scripts/tagging/parseversion.py [hunk 2]
```diff
@@ -109,12 +112,12 @@ def increment( self ):
 
   def __str__( self ):
     """ return version string """
-    versionString = "v%02d-%02d" % ( self.major, self.minor )
+    versionString = "v%02d%s%02d" % ( self.major, self.delimiter, self.minor )
     if self.patch is not None:
-      versionString += "-%02d" % self.patch
+      versionString += ".%02d" % self.patch
```
Comment (andresailer) [inline scripts/tagging/parseversion.py:117]:
```suggestion
      versionString += "%s%02d" % (self.delimiter, self.patch)
```

Comment (fdplacido) [reply scripts/tagging/parseversion.py]: fixed, thanks!

**Review by andresailer — APPROVED** [review]

**Pull request merged**